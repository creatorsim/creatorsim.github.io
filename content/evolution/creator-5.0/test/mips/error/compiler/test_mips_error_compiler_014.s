
#
# Creator (https://creatorsim.github.io/creator/)
#

 .data
 	.align 2
    example:	.directive 221