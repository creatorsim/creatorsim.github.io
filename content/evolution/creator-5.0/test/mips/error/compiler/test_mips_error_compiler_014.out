

[E01] Error: Directive `.directive` isn't defined
   ╭─[ assembly:8:14 ]
   │
 8 │     example:    .directive 221
   │                 ─────┬────  
   │                      ╰────── Unknown directive
───╯

Not executed
keyboard[0x0]:''; display[0x0]:''; 

